packages/form-js-viewer/test/spec/util/GetSchemaVariables.spec.js

```diff
@@ -43,6 +45,22 @@ describe('util/getSchemaVariables', () => {
   });
 
 
+  it('should include variables in root of templates', () => {
+
+    const variables = getSchemaVariables(templateSchema);
+
+    expect(variables).to.eql([ 'myText', 'greeting', 'name', 'showAge', 'age', 'hobbies' ]);
+  });
+
+
+  it('should handle complex template cases', () => {
+
+    const variables = getSchemaVariables(complexTemplateSchema);
+
+    expect(variables).to.eql([ 'value', 'minimum', 'display', 'orgs', 'external1', 'external2', 'parent', 'this' ]);
```

Comment:
True, but it's by design.

For the case where users would provide a context like this:

```
{
	parent: 'Greg',
	this: 'Jeremy',
	....
}
```

There have been cases already where users did this kind of thing, and it broke because of special access characters. We cannot know the context in advance, so we have to kind of assume it's always the case that 'this' is needed in this scenario.

Reply:
If they use `_parent_` or `_this_` however, it will break, and rightfully so because at this point it's deserved :trollface: